src/main/java/com/faforever/api/data/domain/MapPoolAssignment.java

```diff
@@ -0,0 +1,65 @@
+package com.faforever.api.data.domain;
+
+import com.faforever.api.security.elide.permission.WriteMatchmakerMapCheck;
+import com.yahoo.elide.annotation.CreatePermission;
+import com.yahoo.elide.annotation.DeletePermission;
+import com.yahoo.elide.annotation.Include;
+import com.yahoo.elide.annotation.UpdatePermission;
+import lombok.Setter;
+import org.jetbrains.annotations.NotNull;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+
+import javax.persistence.OneToOne;
+import javax.persistence.Table;
+
+
+@Entity
+@Setter
+@Table(name = "map_pool_map_version")
+@Include(rootLevel = true, type = "mapPoolAssignment")
+@CreatePermission(expression = WriteMatchmakerMapCheck.EXPRESSION)
+@UpdatePermission(expression = WriteMatchmakerMapCheck.EXPRESSION)
+@DeletePermission(expression = WriteMatchmakerMapCheck.EXPRESSION)
+public class MapPoolAssignment {
+  private int id;
+  private MapPool mapPool;
+  private MapVersion mapVersion;
+  private int weight;
+  private String mapParams;
```

Comment:
Do we ever care about the contents? Then maybe Map<String, Object> or something like JsonObject

Reply:
Not sure what you mean here. When I try to use those types I just get that it doesn't know how to handle them.

Reply:
Ah yes, Hibernate doesn't support this. Shame. Jackson can handle this.